frontend-instructor/src/components/analytics/Analytics.jsx

```
@@ -0,0 +1,73 @@
+    }
+  }, []);
+  const { courseId } = useParams();
+  return (
+    <div className='px-4'>
+      <div className='border-2 borgray-200 rounded-lg p-4 my-4 flex justify-between items-center'>
+        <div>
+          <h1 className='text-5xl font-bold'>{courseData.title}</h1>
+          <p className='text-gray-500'>{courseData.description}</p>
+        </div>
+        <div className='w-1/5'>
+          <img src='https://2ality.com/2011/10/logo-js/js.jpg' />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)